src/schema/v2/startIdentityVerificationMutation.ts

```diff
@@ -0,0 +1,95 @@
+import { mutationWithClientMutationId } from "graphql-relay"
+import { ResolverContext } from "types/graphql"
+import {
+  GraphQLInputObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+  GraphQLObjectType,
+  GraphQLNonNull,
+} from "graphql"
+import {
+  GravityMutationErrorType,
+  formatGravityError,
+} from "lib/gravityErrorHandler"
+
+const InputType = new GraphQLInputObjectType({
+  name: "StartIdentityVerificationInput",
+  fields: {
+    identityVerificationId: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Primary ID of the identity verification to be started",
+    },
+  },
+})
+
+const FailureType = new GraphQLObjectType<any, ResolverContext>({
+  name: "StartIdentityVerificationFailure",
+  isTypeOf: data => {
+    return data._type === "GravityMutationError"
+  },
+  fields: () => ({
+    mutationError: {
+      type: GravityMutationErrorType,
+      resolve: err => err,
+    },
+  }),
+})
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "StartIdentityVerificationSuccess",
+  isTypeOf: data => data.identity_verification_id,
+  fields: () => ({
+    identityVerificationId: {
+      type: GraphQLString,
+      description: "Primary ID of the started identity verification",
+      resolve: res => res.identity_verification_id,
+    },
+    identityVerificationWizardUrl: {
+      type: GraphQLString,
+      description:
+        "URL that hosts the user-facing identity verification wizard",
+      resolve: res => res.identity_verification_wizard_url,
+    },
+  }),
+})
+
+const OutputType = new GraphQLUnionType({
+  name: "StartIdentityVerificationResponseOrError",
+  types: [SuccessType, FailureType],
+})
+
+export const startIdentityVerificationMutation = mutationWithClientMutationId<
+  any,
+  any,
```

Comment:
Are these types intended to be `any`s?

Reply:
These are straight-up copy+pastes from other mutation definitions. I've mostly been using [`createAccountRequestMutation`](https://github.com/artsy/metaphysics/blob/master/src/schema/v2/createAccountRequestMutation.ts#L88-L92) and [`deleteCreditCardMutation`](https://github.com/artsy/metaphysics/blob/master/src/schema/v2/me/delete_credit_card_mutation.ts#L7-L11).

I briefly dived into type-hell attempting to understand the "why" behind these type arguments, but came back with no more knowledge.

If you have any guidance here, I'm all ears!